renderer/src/hooks/StationModules.tsx

```diff
@@ -1,24 +1,74 @@
 import { useEffect, useState } from 'react'
-import { mockData } from 'src/pages/modules/mockData'
+import * as Name from 'w3name'
 
-// Using mock data
-async function getAllModules () {
-  return mockData
+/**
+ * @returns {Promise<string>} The contract address of the Spark module
+ */
+async function getSparkContractAddress () {
+  const name = Name.parse(
+    'k51qzi5uqu5dmaqrefqazad0ca8b24fb79zlacfjw2awdt5gjf2cr6jto5jyqe'
+  )
+  const revision = await Name.resolve(name)
+  const address = revision.value.split('\n').filter(Boolean).pop()
+  if (typeof address !== 'string') {
+    throw new Error('Failed to resolve Spark contract address')
+  }
+  return address
 }
 
-type Module = typeof mockData[number]
+/* eslint-disable max-len */
+const modules = {
+  spark: {
+    name: 'Spark',
+    description: 'With its array of tools, resources, and support, Spark empowers users to spark new ideas, projects, and solutions that harness the full potential of Filecoin\'s decentralized storage protocol.',
+    contractAddress: '0x',
+    logo: 'https://assets-global.website-files.com/633bfdb71d8e552b70ceff10/633eb29c75502c57fa5ff646_station-download.svg',
```

Comment:
can we inline this svg? Do you think it's a good idea for a hook to import and expose an svg?

Reply:
Yes, we could inline them 👍